packages/react-instantsearch/src/components/Panel.js

```diff
@@ -20,11 +20,16 @@ class Panel extends Component {
 
   constructor(props) {
     super(props);
-    this.state = { canRefine: true };
-    this.canRefine = canRefine => {
-      this.setState({ canRefine });
+
+    this.state = {
+      canRefine: true,
     };
   }
+
+  canRefine = canRefine => {
+    this.setState({ canRefine });
```

Comment:
maybe inline instead of a block? no big opinion on it

Reply:
On a `class` I prefer to use block. It's a bit more readable, not a big opinion neither.